frontend/src/metabase/dashboard/components/EmbedMenu/AdminEmbedMenu.tsx

```diff
@@ -69,15 +69,13 @@ export const AdminEmbedMenu = ({
           onClick={() => setMenuMode("public-link-popover")}
         >
           {isPublicSharingEnabled ? (
-            <Title c="inherit" order={4}>
+            <Title order={4}>
               {hasPublicLink ? t`Public link` : t`Create a public link`}
             </Title>
           ) : (
             <Stack spacing="xs">
-              <Title c="inherit" order={4}>
```

Comment:
The `inherit` property isn't needed here, it was needed before for some coloring reasons but it seems to work fine without it now